component/graph.go

```diff
@@ -0,0 +1,17 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package component // import "go.opentelemetry.io/collector/component"
+
+// Graph is a struct representing the pipelines
+// currently registered with the `Host`.
+type Graph struct {
```

Comment:
Please don't expose any new public API about our internal graph/pipelines.

Reply:
OK, I'll expose it internally for now.